src/com/example/m02/Task45.java

```diff
@@ -0,0 +1,21 @@
+package com.example.m02;
+
+import org.jetbrains.annotations.NotNull;
+
+public class Task45 {
+    public static void main(String[] args) {
+        System.out.println("[null]: " + makeHalfOfName(null));
+        System.out.println("[empty name]: " + makeHalfOfName(""));
+        System.out.println("Magicoraxi: " + makeHalfOfName("Magicoraxi"));
+        System.out.println("Javametro: " + makeHalfOfName("Javametro"));
+    }
+
+    public static String makeHalfOfName(String name) {
+        if (name == null || name.isEmpty()) {
+            return "";
+        }
+        int lastIndex = (int) Math.ceil(name.length() / 2.0);
```

Comment:
please refactor without using Math utils. also you can use simple `int 2` instead of `double 2.0`

Reply:
Solution
if (name == null || name.isEmpty()) {
            return "";
        }
        int lastIndex = name.length() / 2 + (name.length() % 2 == 0 ? 0 : 1);
        return name.substring(0, lastIndex);